docs/REQUIREMENTS.md

```diff
@@ -56,57 +56,60 @@ In order to leverage vSphere, the following items are required for use in your t
 |vsphere_datastore | Name of the vSphere data store to use for the VMs |
 |vsphere_resource_pool | Name of the vSphere resource pool to use for the VMs |
 |vsphere_folder | Name of the vSphere folder to store the VMs |
-|vsphere_template | Name of the VM template to clone to create VMs for the cluster |
+|vsphere_template | Name of the VM template to use to create VMs for the cluster |
 |vsphere_network | Name of the vSphere network |
 
 #### Machine Template Requirements
 
-The current repository supports the provisioning of vSphere VMs if all the following are true:
+The current repository supports the provisioning of vSphere VMs. The following table describes storage-related requirements:
 
 | Requirement | Description |
 | --- | --- |
-| Disk | The `root` partition `/` must be on `/dev/sd2` |
-| Hard Disk | Specify `Thin Provision` to adjust the size of the disk to match the machine requirements listed previously |
+| Disk | The `root` partition `/` must be on `/dev/sd2`. |
+| Hard Disk | Specify `Thin Provision` to adjust the size of the disk to match the machine requirements that were listed previously. |
 
-### Bare-Metal
+### Physical Machines or Linux VMs
 
-For bare-metal provisioning, you must set up ALL systems with the required elements listed previously. These systems must have full network access to each other.
+In order to provision physical machines for a SAS Viya deployment, you must set up ALL systems with the required elements that were listed previously. These systems must have full network access to each other.
 
 ## Network
 
-All systems need routable connectivity to each other.
+All systems require routable connectivity to each other.
 
 ### CIDR Block
 
-The CIDR block for your infrastructure must be able to handle at least the number machines described previously, as well as the virtual IP address that is used for the cluster entrypoint and the cloud provider IP address source range that is needed to support the LoadBalancer services that are created.
+The CIDR block for your infrastructure must be able to accommodate at least the number of machines described in [Machines](#machines). In addition, the CIDR block must have the following:
```

Comment:
I am not sure now with the new physical and bare metal machine conversation that the Machines and/or #machines tag are relevant here now. I believe this needs to be changed.

Reply:
That particular heading needs to stay as "Machines" because it contains subsections for physical/VMs and VMware. So the cross-reference should work.

Reply:
Got it. Thx